api/institutions/views.py

```diff
@@ -0,0 +1,164 @@
+from rest_framework import generics
+from rest_framework import permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from modularodm import Q
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from website.models import Institution, Node, User
+
+from api.base import permissions as base_permissions
+from api.base.filters import ODMFilterMixin
+from api.base.views import JSONAPIBaseView
+from api.base.utils import get_object_or_error
+from api.nodes.views import NodeMixin
+from api.nodes.serializers import NodeSerializer, NodeDetailSerializer
+from api.nodes.permissions import ContributorOrPublic
+from api.users.serializers import UserSerializer, UserDetailSerializer
+
+from .serializers import InstitutionSerializer, InstitutionDetailSerializer
+
+
+class InstitutionList(JSONAPIBaseView, generics.ListAPIView):
```

Comment:
Normally these views would have documentation for how the endpoints work.  Are we going to be advertising this capability right now?